bundle-workflow/src/manifests/bundle_manifest.py

```diff
@@ -83,7 +83,7 @@ def from_s3(bucket_name, build_id, opensearch_version, architecture, work_dir=No
             build_id, opensearch_version, architecture
         )
         S3Bucket(bucket_name).download_file(manifest_s3_path, work_dir)
-        with open("manifest.yml", "r") as file:
+        with open(work_dir + "/manifest.yml", "r") as file:
```

Comment:
Should be `os.path.join()`

Actually change this to be `BundleManifest.from_path(os.path.join(work_dir, 'manifest.yml'))`

Reply:
One more thing. This whole `download_file` business should be 

1. Download to a _temporary directory/file name_.
2. Load via `from_path`.

This way it doesn't pollute current directory from where it's called.

Reply:
Will create a follow issue for this.